portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,8 +68,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // format date
```

Comment:
see my comment in the other PR about comment formatting. Prefer

// Format date.